app.cr

```diff
@@ -27,24 +27,24 @@ def filter(repos, filter)
 end
 
 def fetch_sort(context)
-  sort = context.params["sort"]? || "stars"
+  sort = context.params["sort"]?.try(&.to_s) || "stars"
```

Comment:
@zamith Yes, now `params[option]` returns the union type. Checkout [param_parser.cr#7](https://github.com/sdogruyol/kemal/blob/64fac62804c5c232dbacad148b28a1d31bc4cbdc/src/kemal/param_parser.cr#L7).